src/main/resources/application.properties

```diff
@@ -1,7 +1,7 @@
 spring.jpa.hibernate.ddl-auto=none
 spring.datasource.url=jdbc:mysql://localhost:3306/online_store
-spring.datasource.username=${MYSQL_USERNAME}
-spring.datasource.password=${MYSQL_PASSWORD}
+spring.datasource.username=root
```

Comment:
ti am scris pe what s up astea trebuie sa ramana ${MYSQL_USERNAME} respectiv ${MYSQL_PASSWORD} nu le mai modificam mereu in application.properties trebuie stocate local in intelij, ti.am trimis cu sa adaugi variabilele local